### PR TITLE
support multi ids in sales account and business location

### PR DESCRIPTION
**Pull request by akumari28**
@vendasta/external-apis

**TODO**:
- [x] Review the [pre release checklist](https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks)
Screenshot 2024-02-13 at 7 24 17 PM
Screenshot 2024-02-13 at 7 22 36 PM

### CODE REVIEW

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -314,6 +314,11 @@ paths:
           in: query
           name: 'fields[businessLocations]'
           description: 'Filter businessLocations by the following fields: name, phoneNumbers, customerIdentifier, geoCoordinate, serviceAreaBusiness, hours, and updatedAt. These fields will be included in the response.'
+        - schema:
```
Comment (ks-pai) [inline openapi/platform/platform.yaml:317]: This should be array of strings ?

File: openapi/platform/platform.yaml [hunk 2]
```diff
@@ -314,6 +314,12 @@ paths:
...
+            type: array
```
Comment (chavanrohit123) [inline openapi/platform/platform.yaml:318]:
```suggestion
            type: array
            item:
                type:string
            
```
you can make arrayp[string] like this

**Review by ks-pai — APPROVED** [review]
LGTM

**Review by arunrajkhumar — APPROVED** [review]
LGTM

**Pull request merged**